README.md

```diff
@@ -21,11 +21,11 @@ gems:
 
 ## Usage
 
-The plugin will automatically generate a RSS feed at `/feed/`. No configuration necessary.
+The plugin will automatically generate an RSS feed at `/feed.xml`. No configuration necessary.
 
 ## Contributing
 
-1. Fork it ( https://github.com/jekyll/jekyll-rss-feed/fork )
```

Comment:
BTW, the reason I removed these spaces is that even though (I assume) they were added to match the space the other brackets have, that space is `margin` set via CSS.